scripts/yyParticle.js

```diff
@@ -1459,6 +1713,85 @@ function	ParticleSystem_Particles_Count( _ps )
 }
 
 
+function ParticleSystem_Create_GetLayer(_layerID)
+{
+	var pPartEl = null;
+
+	if (g_isZeus)

```

Comment:
g_isZeus checks are redundant now - g_isZeus is an old check for running on 1.x, you don't need to add these

Reply:
Ah I see, I just kept what was already in there, but I can delete it if it's not needed anymore